slider/replay.py

```diff
@@ -576,6 +636,62 @@ def parse(cls, data, *, library=None, client=None, save=False):
             **mod_kwargs,
         )
 
+    @lazyval
+    def hits(self):
+        b = self.beatmap
```

Comment:
ah sorry that was left over from when i wasn't planning on including this in the library, sure thing.